### PR TITLE
[release/3.0] Issue #780 UserManagedCache docs improvements

### PR DESCRIPTION
**Pull request by ljacomet**

### CODE REVIEW

**Comment by ljacomet** [comment]
test this please

File: docs/src/docs/asciidoc/user/usermanaged.adoc [hunk 1]
```diff
@@ -56,88 +56,35 @@ include::{sourcedir}/impl/src/test/java/org/ehcache/docs/UserManagedCaches.java[
...
-[source,java,indent=0]
-----
-include::{sourcedir}/impl/src/test/java/org/ehcache/docs/UserManagedCaches.java[tag=userManagedEvictionAdvisorCache]
-----
-<1> If you want to hint the eviction algorithm to advise against the eviction of some mappings, you have to
-    configure an instance of `EvictionAdvisor`.
+=== Example with cache event listeners
 
-=== Cache with Listeners example
+If you want to use cache event listeners, you need to provide the executor services with which the events are to be fired.
```
Comment (GaryWKeim) [inline docs/src/docs/asciidoc/user/usermanaged.adoc]: provide the executor service which events are to be fired.

Comment (ljacomet) [inline docs/src/docs/asciidoc/user/usermanaged.adoc]: Not sure I get your update Gary. The executor service provided will be the one firing the events. Is your proposed sentence meaning that?

Comment (GaryWKeim) [inline docs/src/docs/asciidoc/user/usermanaged.adoc]: 1) should executor service be plural here?
2) this fragment: "with which the events are to be fired" doesn't feel natural
3) nothing about the actual content here

**Comment by ljacomet** [comment]
From the remarks made by @GaryWKeim I ended up expanding the cache event listeners section.

**Pull request merged**